### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by hrshdhgd**
- Creates `example.md` and `example.yaml` files.
 - Uses `examples.yaml` as an input to run all set commands.

### CODE REVIEW

File: tests/test_case_dumpers.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    EdgeCreation,
+    PlaceUnder,
+    RemoveUnder,
+    EdgeDeletion,
+    NodeDeepening,
+)
+import kgcl_schema.grammar.parser as kgcl_parser
+from kgcl_schema.utils import from_yaml
+from tests import TMP_YAML
+from tests.cases import CASES
```
Comment (cmungall) [inline tests/test_case_dumpers.py:22]: did you mean to check in this file?

File: src/docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# KGCL Commands.
+
+## `NodeRename`:
+	Command using CURIEs: `rename GO:0005635 from 'nuclear envelope' to 'foo bar'`
```
Comment (cmungall) [inline src/docs/examples.md:4]: did you mean to include the leading whitespace? This means quote as code in yaml

File: src/docs/examples.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	Command using URIs: `rename http://purl.obolibrary.org/obo/GO_0005635 from 'nuclear envelope' to 'foo bar'`
```
Comment (cmungall) [inline src/docs/examples.md:5]: I don't think seeing the equivalent with URIs is useful. Let's remove this

File: src/docs/examples.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# KGCL Commands.
+
+## `NodeRename`:
```
Comment (cmungall) [inline src/docs/examples.md:3]: Link to URIs like https://w3id.org/kgcl/NodeRename

File: src/docs/examples.md (hunk 2 above)
Comment (hrshdhgd) [reply src/docs/examples.md:4]: This is markdown. YAML is separate.

**Review by cmungall — CHANGES_REQUESTED** [review]
This is the format I had in mind:

```
## Example: basic renaming

Command: `rename XX:123 to 'xxx'`

YAML:

<yaml block>

```
```

File: src/docs/examples.md (hunk 4 above)
Comment (hrshdhgd) [reply src/docs/examples.md]: Done

**Pull request merged**